web/app/themes/mitlib-moh/js/3play.hack.js

```diff
@@ -0,0 +1,19 @@
+$(window).load(function(){
+	// $('#collection form').on('submit', function(){
```

Comment:
Is there a reason we wanted to leave these commented out in the code base? If they are useful examples we will need/want later can we mark them as such?

Reply:
I'm not sure, to be honest - I didn't go deep enough in this work to look for commented-out code blocks. Looking at it now, I'm not sure whether these are calls that should be preserved or not. We won't be able to dig in on this in context until there is content in the site for this to work against - so my preference would probably be to make a ticket for something like this, to evaluate later in the project.

Reply:
Yeah that makes sense (not even worrying about the ticket for now). It slipped my mind that we had agreed to minimize the changes at this stage.